docs/hacking/install.rst

```diff
@@ -131,15 +116,15 @@ system that Docker can be installed on:
 
    .. code-block:: bash
 
-      docker run -d --name nsqd -p 4150:4150 -p 4151:4151 nsqio/nsq /nsqd
-      docker run -d --name postgres -p 5432:5432 postgres
-      docker run -d --name redis -p 6379:6379 redis
-      docker run -d --name elasticsearch -p 9200:9200 -p 9300:9300 nickstenning/elasticsearch-icu
+      sudo docker run -d --name nsqd -p 4150:4150 -p 4151:4151 nsqio/nsq /nsqd
```

Comment:
Ah, on linux these commands will fail without sudo unless you've taken additional steps, so I [added sudo to make it consistent with other commands in these docs that require sudo](https://github.com/hypothesis/h/pull/3219/commits/8624a4bbc11f49e7c5dc5a82e55c1b8df703b06f). If you don't then the commands fail with an error message that doesn't really suggest that it's a sudo / permissions thing. This is an OS X vs Linux difference then. We could remove the sudos, and add half a sentence somewhere to say "install docker and if necessary (e.g. on linux) configure it to run without sudo"?